### DISCUSSION TITLE
Error using SOLWEIG

### DISCUSSION THREAD

**Discussion by TigerMINATO** [UMEP-dev/UMEP#292] [Q&A] [answered]
Hi, I'm a student in Japan. I'm sorry for my bad English.
I got an error when using SOLWEIG, and I don't understand, so I came to ask a question.
Here is the error message. "DEM and DSM were raised unequally (difference>0.5m)"
This error only appears on my desktop computer. I am using the same data on my laptop and it does not have this error and works fine. The data that was distributed in the tutorial works on my desktop computer.
What is the problem?

By the way, the desktop computer is made by NEC. My laptop is made by Dell. I have installed Pandas on both of them.

**Comment by biglimp** [comment, marked as answer]
Do you have the same versions of UMEP on both computers?

**Reply by TigerMINATO** [reply]
Yes, I am using QGIS with version 3.14 for both.

**Reply by biglimp** [reply]
Not version of QGIS but version of UMEP

**Reply by TigerMINATO** [reply]
Sorry, at first I was using the same version 3.15.4 for both. However, I got an error on my desktop computer, so I am now using version 3.17.3 on my desktop computer. The laptop has always been on version 3.15.4

**Reply by biglimp** [reply]
Try update on both and see if error persists

**Reply by TigerMINATO** [reply]
Thank you very much. Anyway, I'll try that next Monday.

**Reply by TigerMINATO** [reply]
Today, I updated UMEP to the latest version, which is the same for both. However, I got the same error on my laptop.

**Reply by biglimp** [reply]
If you have smae versions this error should not appear. I have no clue why. Try starting QGIS with an empty project, add your data and try again. You can also attach your DSM and DEM for me to have a llok at.

**Reply by TigerMINATO** [reply]
I tried that too, but it didn't work.
I will send you the DSM and DEM in a zip file, please check it out.
[dsm2.zip](https://github.com/UMEP-dev/UMEP/files/6685766/dsm2.zip)
[dem2.zip](https://github.com/UMEP-dev/UMEP/files/6685768/dem2.zip)

**Reply by biglimp** [reply]
You have some negative values in your datasets. remove those (e.g. reclassify to 0) and try again.

**Reply by TigerMINATO** [reply]
OK,I'll try it.

**Reply by TigerMINATO** [reply]
Thank you very much. It was caused by negative values in the data. Thanks to you, SOLWEIG worked fine and I was able to calculate.
But why was the older version able to calculate even if there were negative values?

**Reply by biglimp** [reply]
Looking at the code #230 is the reason why the control of differences in min values was included.

**Reply by TigerMINATO** [reply]
Well, thank you for your help.
